### PR TITLE
Refactoring td agent role

### PR DESCRIPTION
**Pull request by tei-k**
# Description
https://scalar-labs.atlassian.net/browse/DLT-6567

# Done
- Fix to use package module to install td-agent
- Create `conf.d` directory to include `*.conf`

### CODE REVIEW

File: provision/ansible/playbooks/bastion-server.yml [hunk 1]
```diff
@@ -8,7 +8,11 @@
     - role: td-agent
       when: enable_tdagent is undefined or enable_tdagent == '1'
   vars:
-    tdagent_conf_template: templates/bastion-server/td-agent.conf.j2
+    tdagent_confd_templates:
+      - {
+          src: "templates/td-agent-confd/td-agent-bastion.conf.j2",
+          dest: "bastion.conf",
+        }
```
Comment (ymorimo) [inline provision/ansible/playbooks/bastion-server.yml:15]: This is the same as the following, but any reason to format like this?

```yaml
    tdagent_confd_templates:
      - src: "templates/td-agent-confd/td-agent-bastion.conf.j2"
        dest: "bastion.conf"
```

Comment (tei-k) [reply provision/ansible/playbooks/bastion-server.yml]: I just want to write in one line, But automatically formatted by my vscode plugin. 😓 
```
tdagent_confd_templates:
  - { src: "templates/td-agent-confd/td-agent-bastion.conf.j2", dest: "bastion.conf" }
```

Fixed in oneline (9f55901) How about this?

Comment (ymorimo) [reply provision/ansible/playbooks/bastion-server.yml]: Looks good, but isn't it likely to be formatted again?

Comment (feeblefakie) [reply provision/ansible/playbooks/bastion-server.yml]: I think what @ymorimo suggested [here](https://github.com/scalar-labs/scalar-terraform/pull/164#discussion_r454059067) is more common format of yaml.
@tei-k Can we do just make it like that ?

Comment (tei-k) [reply provision/ansible/playbooks/bastion-server.yml]: Just disabled my plugin.
refs:
https://github.com/lrk/ansible-role-tdagent

Comment (feeblefakie) [reply provision/ansible/playbooks/bastion-server.yml]: Hmm, not sure if it's good to disable that...

**Pull request merged**